test/util.dart

```diff
@@ -54,6 +54,12 @@ Map unmodifiableMap([Map map1, Map map2, Map map3, Map map4]) {
   return new Map.unmodifiable(merged);
 }
 
+bool isDDC() {
```

Comment:
Same comment as in the other PR, this should be called `assertsEnabled` or something

Reply:
Addressed in https://github.com/cleandart/react-dart/pull/258/commits/893bc4f052c20b519dcc28626e0895146c9596b1